Sources/SwiftInspectorVisitors/Tests/NestableTypeVisitorSpec.swift

```diff
@@ -1002,6 +1005,12 @@ final class NestableTypeVisitorSpec: QuickSpec {
           expect(property?.name) == "someFooFooClassProperty"
         }
 
+        it("finds the function declared in FooEnum.FooFooClass") {
```

Comment:
we don't _need_ this test but it's good to have it – we already have a similar test for properties above